opentasks/src/main/res/values-cy/strings.xml

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Manifest stuff -->
+    <string name="app_name">OpenTasks</string>
+
+    <!-- Task list -->
+    <string name="menu_add_task">Ychwanegu tasg</string>
+    <string name="menu_add_task_condensed">Ychwanegu</string>
+    <string name="menu_show_completed">Dangos tasgau wedi'u cyflawni</string>
```

Comment:
Please ensure you also escape single quotation marks with a `\`. The compiler fails otherwise, see `https://travis-ci.org/dmfs/opentasks/builds/590884490#L2735